projects/data-structures-algorithms-sprint.md

```diff
@@ -28,7 +28,9 @@ This sprint will also help you practice study skills, summarizing, teamwork, and
 ### Schedule
 
 #### Day 1
-In the morning, the Program Facilitator will give a quick introduction to data structures, split you into groups, and assign two data structures and three algorithms topics per group. Homework: read about your assigned data structures topics, highlight or outline the most important parts, and review runtime complexity ([topic outline](https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md), [slides](https://drive.google.com/open?id=1ZcOdekB_aP59huZdp4X0u6EfUJKgxzK7y8LqCmzSLC8), [video](https://drive.google.com/open?id=1ZoHxJMUiKOKPqu69vX3b_aeYGlDlRL6n)). 
+In the morning, the Program Facilitator will give a quick introduction to data structures, split you into groups, and assign two data structures and three algorithms topics per group.The group members compare notes about topics, start creating introductory slides, and decide who will present what. groups will plan out demos, select beginner exercises and resources, polish slides, and practice presenting their introductory topic overviews (remember that reading is discouraged and eye contact and a loud, confident voice are encouraged).
+
+ Homework: read about your assigned data structures topics, highlight or outline the most important parts, and review runtime complexity ([topic outline](https://github.com/Techtonica/curriculum/blob/master/runtime-complexity/runtime-complexity.md), [slides](https://drive.google.com/open?id=1ZcOdekB_aP59huZdp4X0u6EfUJKgxzK7y8LqCmzSLC8), [video](https://drive.google.com/open?id=1ZoHxJMUiKOKPqu69vX3b_aeYGlDlRL6n)). 
```

Comment:
@yati1998 The slideshow is not following the formatting guidelines.
1. On slide no. 4 & 16 there's no bar and Techtonica logo at the bottom.
2. Please make sure all text is in `Lato` font style, all the headers font size is 28 and all the regular text size is either 18 or 24.

@alodahl Please give me edit access to that slideshow so that I can review all the other guidelines.

Reply:
@kundan28 according to the issue, her task was to fix links in prerequisite sections; slideshows are not involved.

Reply:
she's simply fixing this file to match master.

Reply:
Okay.